src/opts.rs

```diff
@@ -11,6 +11,15 @@ pub struct Opts {
 	#[clap(short, long)]
 	pub template: PathBuf,
 
+	/// This flag tells the command to parse all templates found in the same
+	/// path where the given template is located.
+	#[clap(short, long)]
```

Comment:
I am thinking the following may help:

```suggestion
	#[clap(short, long, alias="inherit")]
```
what do you think ?

That would make the inheritance example look like:
```
tera -t ./data/inheritance/child.txt --inherit --env-only
```